### PR TITLE
feat(client): cli supports displaying eval on locally served web pages

### PR DESCRIPTION
**Pull request by jialeicui**
## Description

- Support showing eval detail via the web browser
- Support saving the panel layout setting to the `workdir/.starwhale/panel.json` (mp build cmd will use this file)
- Hide all the sidebar and header in the web page

https://user-images.githubusercontent.com/3217223/224913096-b1d19a7c-894d-4224-b215-6e17fc9b571a.mp4

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/web/project.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+@router.get("/{project_id}")
+def get_project(project_id: str) -> SuccessResp:
+    return success(project)
+
+
+@router.get("/{project_id}/job/{job}")
+def get_job(project_id: str, job: str) -> SuccessResp:
+    return success({"id": 1, "uuid": job, "modelName": "mock-model"})
```
Comment (tianweidut) [inline client/starwhale/web/project.py:26]: `1` -> `"1"` ?

**Comment by tianweidut** [comment]
@jialeicui  does the released python package include console js files?

**Review by waynelwz — APPROVED** [review]
LGTM

File: client/starwhale/web/project.py (hunk 1 above)
Comment (jialeicui) [reply client/starwhale/web/project.py]: fixed

**Comment by jialeicui** [comment]
> @jialeicui does the released python package include console js files?

Yes, you're right :+1:  
I will make another PR to do this 
@tianweidut

**Review by tianweidut — APPROVED** [review]

**Pull request merged**